commitlint.config.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+    extends: ["@commitlint/config-conventional"],
+    rules: {
+        "body-case": [2, "always", "sentence-case"],
+        "body-max-line-length": [2, "always", 72],
+        "header-max-length": [2, "always", 72],
+        "footer-max-length": [2, "always", 52],
+        "type-enum": [2, "always", getTypes()],
+        "issue-id-required": [2, "always"],
+    },
+    plugins: [
+        {
+            rules: {
+                "issue-id-required": ({ subject }) => {
+                    const issueIdRegex = /\s#(\d+)$/;
+                    if (!issueIdRegex.test(subject)) {
+                        return [false, "Issue ID is required at the end of the commit message"];
+                    }
+                    return [true];
+                },
+            },
+        },
```

Comment:
We could use a footer in the commit message every time to specify the issue like
```
feat: add something

Refs: #1234
```

Reply:
i added that plugin because our contributing convention defines commit messages as 
```
<type>[optional scope][!]: <description> #<issue-id>
[optional body]
[optional footer(s)]
```
it says that issue id should be included at the end of description.

Reply:
Yeah, it's fine. I just wanted to share an alternative with you.